tests/containers/isolation.pm

```diff
@@ -0,0 +1,100 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: isolation
+# Summary: Test container network isolation
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
+use containers::common qw(install_packages);
+use utils qw(script_retry systemctl);
+
+my $runtime;
+my $network = "test_internal_net";
```

Comment:
```suggestion
my $network = "test_isolated_network";
```
Let's use a less generic name to avoid possible name collisions with other test runs.

Not that they are likely to happen, but still it's a good thing to do I think.

Reply:
Fixed